### PR TITLE
Feature/llvm to 3.11

### PR DESCRIPTION
**Pull request by KVS85**
See description in oskar PR: https://github.com/arangodb/oskar/pull/585

### CODE REVIEW

File: VERSIONS [hunk 1]
```diff
@@ -1,13 +1,14 @@
 CXX_STANDARD "20"
 STARTER_REV "v0.18.6-preview-4"
 SYNCER_REV "v2.19.7"
-GCC_LINUX "11.2.1_git20220219-r2"
+GCC_LINUX "13.2.0"
```
Comment (jsteemann) [inline VERSIONS:4]: Why are we changing the version of gcc here as well?
I am not sure if that is used anywhere later in the toolchain, but I am pretty sure that compiling 3.11 with g++-13.2.0 will not work.

**Review by neunhoef — APPROVED** [review]
LGTM

File: VERSIONS (hunk 1 above)
Comment (KVS85) [reply VERSIONS]: Removed.

**Comment by KVS85** [comment]
- [ ] https://jenkins.arangodb.biz/view/PR/job/arangodb-matrix-pr/32838/

**Pull request merged**